applications-dropins/qvm-open-in-dvm.desktop

```diff
@@ -0,0 +1,10 @@
+### Note: With this installed, typing "xdg-settings set default-web-browser qvm-open-in-dvm.desktop" will make it so that in gnome-terminal
```

Comment:
This directory is for overrides of third-party menu entries. Put this one in the `app-menu` dir.